doc/developer/deprecations.rst

```diff
@@ -49,3 +49,4 @@ Version 3.0
 * In ``networkx/utils/misc.py`` remove ``is_string_like`` and related tests.
 * In ``networkx/utils/misc.py`` remove ``make_str`` and related tests.
 * Remove ``networkx/utils/contextmanagers.py`` and related tests.
+* In ``networkx/drawing/nx_agraph.py`` remove ``display_pygraphviz`` and related tests.
```

Comment:
I realized that I started the pattern of prefixing things w/ ``networkx`` in #4162.  I started removing the prefix there.  You should do it here as well.